js/data.js

```diff
@@ -52,4 +59,6 @@ function generateTestData (numberOfData = 25) {
     testData
   };
 }
-export {generateTestData};
+
+const newTestData = generateTestData(25);
+export {newTestData};
```

Comment:
```suggestion
export {generateTestData};
```
у тебя вся логика старта приложения должна быть в main.js
там не должны быть просто импорты. там должны быть импорты и вызовы функций